File: computercraft/archive/try_refuel.py
from .constants import Mock

turtle = Mock.Turtle()


def try_refuel(offset_amount: int) -> bool:
    pass  # TODO: Add logic for refueling

    if turtle.getFuelLevel() < offset_amount * 2:  # Expecting at least 50% efficiency
        return False
    else:
        return True

File: mining/try_refuel.lua
function try_refuel(offset_amount)
    -- TODO: Add logic for refueling

    --[[
    Below calc assumes that you want to return to where you were before backtracking to refuel,
    and be able to make at least 1 more move before needing to backtrack once again
    --]]
    if turtle.getFuelLevel() < (offset_amount * 2) + 2 then
        return False
    else
        return True
    end
end


return try_refuel
